test/src/api/responderTest.php

```diff
@@ -4676,7 +4676,7 @@ public function test_interface_game_008() {
         array_unshift($expData['gameActionLog'], array('timestamp' => 'TIMESTAMP', 'player' => 'responder003', 'message' => 'responder003 chose to perform a Skill attack using [tm(8):3] against [(8):7]; responder003 must turn down fire dice to complete this attack'));
         $expData['gameActionLogCount'] = 7;
         $expData['gameState'] = 'ADJUST_FIRE_DICE';
-        $expData['validAttackTypeArray'] = array();
+        $expData['validAttackTypeArray'] = array('Skill');
```

Comment:
Fine.  I still don't like overloading `validAttackTypeArray`, but i guess in this case it's fairly unambiguous.  It'll bite us eventually, but hopefully not today.